aspnetcore/index.yml

```diff
@@ -0,0 +1,429 @@
+### YamlMime:Hub
+
+title: ASP.NET documentation
+summary: Learn to use ASP.NET Core to create web apps and services that are fast, secure, cross-platform, and cloud-based. Browse tutorials, sample code, fundamentals, API reference and more.
+brand: aspnet
+
+metadata:
+  title: ASP.NET documentation
+  description: Learn to use ASP.NET Core to create web apps and services that are fast, secure, cross-platform, and cloud-based. Browse tutorials, sample code, fundamentals, API reference and more.
+  ms.product: aspnet
+  ms.topic: hub-page
+  author: WadePickett
+  ms.author: wpickett
+  ms.date: 03/24/2020
+
+# highlightedContent section (optional)
+# Maximum of 8 items
+highlightedContent:
+# itemType: architecture | concept | deploy | download | get-started | how-to-guide | learn | overview | quickstart | reference | tutorial | video | whats-new
+  items:
+    # Card
+    - title: "Create an ASP.NET Core app on any platform in 5 minutes"
+      itemType: get-started
+      url: getting-started/index.md
+    # Card
+    - title: "ASP.NET Core overview"
+      itemType: overview
+      url: introduction-to-aspnet-core.md
+    # Card
+    - title: "ASP.NET Core video tutorials"
```

Comment:
I agree a landing page the hub points to that focuses on community content would be nice.  We used to have  a section like that for years on the https://asp.net site previously called "Community Highlights" I believe.  It takes a little bit of resource commitment to keep up every week though, which may be  part of the reason it disappeared.  Maybe that page should be this one, since several folks leading community focused efforts are involved with it?  https://dotnet.microsoft.com/platform/community  We can discuss that one on a different thread dedicated to it..  Thanks very much for the review and your insights.